bundles/org.openhab.binding.vektiva/src/main/resources/ESH-INF/binding/binding.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<binding:binding id="vektiva" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:binding="https://openhab.org/schemas/binding/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/binding/v1.0.0 https://openhab.org/schemas/binding-1.0.0.xsd">
+
+	<name>Vektiva Binding</name>
+	<description>This is the binding for Vektiva products.</description>
```

Comment:
Consider mentioning here that is for the local-api if you can add it in comprehensible way.

Reply:
mentioned